### PR TITLE
Link to the 0.5 release candidate downloads

### PR DESCRIPTION
**Pull request by ararslan**
I used the 0.4-rc version of this file as a template and I took the download links from https://groups.google.com/forum/#!topic/julia-users/shrnNHpyliA.

Fixes https://github.com/JuliaLang/julia/issues/17881

### CODE REVIEW

**Comment by tkelman** [comment]
We should probably link to rc1+1 rather than rc1, or use the 0.5-latest download link.

**Comment by ararslan** [comment]
What do I have to do to link to rc1+1? Can I just change the URLs to use rc1+1 instead of rc1?

Also where is this fancy 0.5-latest download?

**Comment by tkelman** [comment]
I don't think I moved rc1+1 to a permanent location on julialang, it's on julianightlies but will get cleaned up from there eventually. 0.5-latest should just be replacing 0.5.0-rc1, but check at https://s3.amazonaws.com/julialang

**Comment by ararslan** [comment]
Does 0.5-latest have a corresponding tarball somewhere?

**Comment by tkelman** [comment]
Usually it corresponds to the most recent release candidate or patch release for a given minor series. Right now it's slightly different because of the mistake in tagging rc1 with the major bug in it.

**Comment by ararslan** [comment]
Okay, so for the source links should I leave it as the tagged rc1 since that has the source and source+dependencies tarballs?

**Comment by tkelman** [comment]
may as well

**Comment by ararslan** [comment]
Done. Look okay?

File: downloads/index.md [hunk 1]
```diff
@@ -84,22 +84,22 @@ tarball is much smaller in size, but will download dependency tarballs on demand
...
 </tr>
 <tr>
     <th> Source </th>
-    <td colspan="2"> <a href="https://github.com/JuliaLang/julia/releases/download/v0.5.0-rc1/julia-0.5.0-rc1.tar.gz">Tarball</a> (<a href="https://github.com/JuliaLang/julia/releases/download/v0.5.0-rc1/julia-0.5.0-rc1.tar.gz.asc">GPG</a>) </td>
-    <td colspan="2"> <a href="https://github.com/JuliaLang/julia/releases/download/v0.5.0-rc1/julia-0.5.0-rc1-full.tar.gz">Tarball with dependencies</a> (<a href="https://github.com/JuliaLang/julia/releases/download/v0.5.0-rc1/julia-0.5.0-rc1-full.tar.gz.asc">GPG</a>) </td>
+    <td colspan="2"> <a href="https://github.com/JuliaLang/julia/releases/download/v0.5.0-rc1/julia-0.5.0-rc1-latest.tar.gz">Tarball</a> (<a href="https://github.com/JuliaLang/julia/releases/download/v0.5.0-rc1/julia-0.5.0-rc1-latest.tar.gz.asc">GPG</a>) </td>
+    <td colspan="2"> <a href="https://github.com/JuliaLang/julia/releases/download/v0.5-latest/julia-0.5-latest-full.tar.gz">Tarball with dependencies</a> (<a href="https://github.com/JuliaLang/julia/releases/download/v0.5.0-rc1/julia-0.5.0-rc1-full.tar.gz.asc">GPG</a>) </td>
```
Comment (tkelman) [inline downloads/index.md]: sources don't have an 0.5-latest

File: downloads/index.md [hunk 2]
```diff
@@ -84,22 +84,22 @@ tarball is much smaller in size, but will download dependency tarballs on demand
...
+    <td colspan="6"> <a href="https://s3.amazonaws.com/julialang/bin/osx/x64/0.5/julia-0.5-latest-osx10.7+.dmg">10.7+ 64-bit</a> </td>
 </tr>
 <tr>
     <th> Generic Linux binaries </th>
-    <td colspan="3"> <a href="https://julialang.s3.amazonaws.com/bin/linux/x86/0.5/julia-0.5.0-rc1-linux-i686.tar.gz">32-bit</a> (<a href="https://julialang.s3.amazonaws.com/bin/linux/x86/0.5/julia-0.5.0-rc1-linux-i686.tar.gz.asc">GPG</a>)</td>
-    <td colspan="3"> <a href="https://julialang.s3.amazonaws.com/bin/linux/x64/0.5/julia-0.5.0-rc1-linux-x86_64.tar.gz">64-bit</a> (<a href="https://julialang.s3.amazonaws.com/bin/linux/x64/0.5/julia-0.5.0-rc1-linux-x86_64.tar.gz.asc">GPG</a>)</td>
+    <td colspan="3"> <a href="https://julialang.s3.amazonaws.com/bin/linux/x86/0.5/julia-0.5-latest-linux-i686.tar.gz">32-bit</a> (<a href="https://julialang.s3.amazonaws.com/bin/linux/x86/0.5/julia-0.5-latest-linux-i686.tar.gz.asc">GPG</a>)</td>
+    <td colspan="3"> <a href="https://julialang.s3.amazonaws.com/bin/linux/x64/0.5/julia-0.5-latest-linux-x86_64.tar.gz">64-bit</a> (<a href="https://julialang.s3.amazonaws.com/bin/linux/x64/0.5/julia-0.5-latest-linux-x86_64.tar.gz.asc">GPG</a>)</td>
```
Comment (tkelman) [inline downloads/index.md]: I don't think I gpg-signed rc1+1, but that can wait until rc2 to be sorted out

File: downloads/index.md (hunk 1 above)
Comment (ararslan) [inline downloads/index.md]: Oh whoops, that was an oversight on my part. Will fix.

File: downloads/index.md (hunk 2 above)
Comment (ararslan) [inline downloads/index.md]: Should I remove the GPG links for now then?

**Comment by ViralBShah** [comment]
Should we wait for rc2 before publishing this, assuming rc2 is soon?

**Comment by tkelman** [comment]
yeah probably

**Comment by ararslan** [comment]
What's the ETA on rc2, any idea?

File: downloads/index.md [hunk 3]
```diff
@@ -67,6 +67,43 @@ Older releases of Julia for all platforms are available on the [Older releases p
...
+    <td colspan="3"> <a href="https://s3.amazonaws.com/julialang/bin/winnt/x64/0.5/julia-0.5-latest-win64.exe">64-bit</a> </td>
+</tr>
+<tr>
+    <th> Mac OS X Package (.dmg) </th>
+    <td colspan="6"> <a href="https://s3.amazonaws.com/julialang/bin/osx/x64/0.5/julia-0.5-latest-osx10.7+.dmg">10.7+ 64-bit</a> </td>
+</tr>
+<tr>
+    <th> Generic Linux binaries </th>
+    <td colspan="3"> <a href="https://julialang.s3.amazonaws.com/bin/linux/x86/0.5/julia-0.5-latest-linux-i686.tar.gz">32-bit</a> (<a href="https://julialang.s3.amazonaws.com/bin/linux/x86/0.5/julia-0.5-latest-linux-i686.tar.gz.asc">GPG</a>)</td>
+    <td colspan="3"> <a href="https://julialang.s3.amazonaws.com/bin/linux/x64/0.5/julia-0.5-latest-linux-x86_64.tar.gz">64-bit</a> (<a href="https://julialang.s3.amazonaws.com/bin/linux/x64/0.5/julia-0.5-latest-linux-x86_64.tar.gz.asc">GPG</a>)</td>
```
Comment (tkelman) [inline downloads/index.md]: the `.asc` signatures are tag specific, I don't make `-latest` copies of those

File: downloads/index.md [hunk 4]
```diff
@@ -67,6 +67,43 @@ Older releases of Julia for all platforms are available on the [Older releases p
...
+</tr>
+<tr>
+    <th> Source </th>
+    <td colspan="2"> <a href="https://github.com/JuliaLang/julia/releases/download/v0.5.0-rc2/julia-0.5.0-rc2-latest.tar.gz">Tarball</a> (<a href="https://github.com/JuliaLang/julia/releases/download/v0.5.0-rc2/julia-0.5.0-rc2-latest.tar.gz.asc">GPG</a>) </td>
```
Comment (tkelman) [inline downloads/index.md]: no `-latest` here

File: downloads/index.md [hunk 5]
```diff
@@ -67,6 +67,43 @@ Older releases of Julia for all platforms are available on the [Older releases p
...
+    <td colspan="3"> <a href="https://s3.amazonaws.com/julialang/bin/winnt/x86/0.5/julia-0.5-latest-win32.exe">32-bit</a> </td>
+    <td colspan="3"> <a href="https://s3.amazonaws.com/julialang/bin/winnt/x64/0.5/julia-0.5-latest-win64.exe">64-bit</a> </td>
+</tr>
+<tr>
+    <th> Mac OS X Package (.dmg) </th>
+    <td colspan="6"> <a href="https://s3.amazonaws.com/julialang/bin/osx/x64/0.5/julia-0.5-latest-osx10.7+.dmg">10.7+ 64-bit</a> </td>
+</tr>
+<tr>
+    <th> Generic Linux binaries </th>
+    <td colspan="3"> <a href="https://julialang.s3.amazonaws.com/bin/linux/x86/0.5/julia-0.5-latest-linux-i686.tar.gz">32-bit</a> (<a href="https://julialang.s3.amazonaws.com/bin/linux/x86/0.5/julia-0.5-latest-linux-i686.tar.gz.asc">GPG</a>)</td>
```
Comment (tkelman) [inline downloads/index.md]: no `-latest*.asc`

Comment (ararslan) [inline downloads/index.md]: Sorry, missed that one

**Comment by tkelman** [comment]
Great, thanks for this!

**Comment by ararslan** [comment]
As always, my pleasure! 😄

**Pull request merged**